scripts/docker/yscope-velox-builder.dockerfile

```
@@ -0,0 +1,76 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builder image with dependencies and pre-populated ccache for faster CI builds
+FROM ghcr.io/y-scope/docker-github-actions-runner:ubuntu-jammy
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Copy only scripts first for dependency installation.
+# This optimizes Docker layer caching: changes to other source files won't invalidate
+# the dependency installation layer - it only rebuilds when scripts change.
+COPY scripts /tmp/velox-deps/
+
+# Set timezone to avoid interactive prompts during apt installations
+ENV TZ=Etc/UTC
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install CMake 3.28.3 using official binary installer
+# NOTE: setup-ubuntu.sh installs cmake via pip, but pip-installed cmake is not reliably found
+# in container environments, causing FastFloat library build failures. Using the official
+# binary installer ensures cmake is available and dependencies build correctly in the current
+# CI containerized environment.
+RUN wget --progress=dot:giga https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.sh \
+ && chmod +x cmake-3.28.3-linux-x86_64.sh \
+ && ./cmake-3.28.3-linux-x86_64.sh --skip-license --prefix=/usr/local \
+ && rm cmake-3.28.3-linux-x86_64.sh
```

We should indent by 4, not 2:

```suggestion
   && chmod +x cmake-3.28.3-linux-x86_64.sh \
   && ./cmake-3.28.3-linux-x86_64.sh --skip-license --prefix=/usr/local \
   && rm cmake-3.28.3-linux-x86_64.sh
```